assets/src/components/FeaturesTable.js

```diff
@@ -17,6 +17,12 @@ import { mainLizmap, mainEventDispatcher } from '../modules/Globals.js';
  * @element lizmap-features-table
  * @fires features.table.item.dragged
  * @fires features.table.rendered
+ * <lizmap-features-table draggable="yes"sortingorder="asc" sortingfield="libsquart"
+ *                        withgeometry="1" expressionfilter="quartmno = 'HO'"
+ *                        uniquefield="id" layerid="sousquartiers_24ceec66_e7fe_46a2_b57a_af5c50389649"
+ *                        layertitle="child sub-districts" id="0782b34c-840c-4b0f-821c-1b66c964e371">
+ *      <lizmap-field alias="Âge du quartier" expression="rand(12,56)" description="Libellé de l'âge du quartier"></lizmap-field>
```

Comment:
Can you switch to English for these examples ?

I was wondering how behaves JSDoc for documenting this class with this HTML, if it can be auto-documented for https://docs.3liz.org/lizmap-web-client/js/module-FeaturesTable.html and later when we will let our users to write their HTML Element https://jsdoc.app/